internal/code-generator/generator/drop_whitespace_test.go

```diff
@@ -0,0 +1,29 @@
+package generator
```

Comment:
Is this file here for your own experimentation? I wouldn't generally advise keeping a benchmark for an internal detail like this, especially one that is only for testing.